src/shared/components/GeoPlot.tsx

```diff
@@ -0,0 +1,56 @@
+// Libraries
+import {FunctionComponent} from 'react'
+import {Config, Table} from '@influxdata/giraffe'
+
+// Types
+import {GeoViewProperties} from 'src/types'
+import {getLatLon} from '../utils/vis'
+
+interface Props {
+  children: (config: Config) => JSX.Element
+  fluxGroupKeyUnion?: string[]
+  table: Table
+  viewProperties: GeoViewProperties
+}
+
+const GeoPlot: FunctionComponent<Props> = ({
+  children,
+  table,
+  viewProperties,
+}) => {
+  const {
+    layers,
+    zoom,
+    allowPanAndZoom,
+    detectCoordinateFields,
+    mapStyle,
+  } = viewProperties
+  const tileServerConfiguration = {
+    tileServerUrl:
+      'https://api.mapbox.com/styles/v1/influxdata/ckhl79okh00o919npquotuqxp/tiles/256/{z}/{x}/{y}@2x?access_token=pk.eyJ1IjoiaW5mbHV4ZGF0YSIsImEiOiJja2hndnM0eWQwNTVtMzRwZnd5cmQxNHhoIn0.IEAjIguiM67ql4mhiD88rw',
```

Comment:
what's up with this being hardcoded here?

Reply:
good catch, i removed it, then rebased and pushed and forgot to check